media-gfx/noteshrink/noteshrink-0.1.1.recipe

```diff
@@ -3,28 +3,55 @@ DESCRIPTION="Noteshrink is an utility that is used to convert scans of notes to
 HOMEPAGE="https://github.com/mzucker/noteshrink"
 COPYRIGHT="2016 Matt Zucker"
 LICENSE="MIT"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="https://github.com/mzucker/noteshrink/archive/$portVersion.tar.gz"
 CHECKSUM_SHA256="2a29c09768973e688b693b61337c6c49384e8123cf88824dba335cc8c4ed2ca8"
 
-ARCHITECTURES="!any"
-# Doesn't work on gcc2hybrid - missing dependencies.
+ARCHITECTURES="any"
+# The package is "any" because the result is only a Python script under /bin/.
+# However, on gcc2 systems, it will depend on Python modules only available for _x86.
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	hostArchSuffix="_x86"
+else
+	hostArchSuffix="$secondaryArchSuffix"
+fi
 
 PROVIDES="
 	noteshrink = $portVersion
 	cmd:noteshrink
 	"
 REQUIRES="
-	haiku
-	imagemagick
-	numpy_python >= 1.10
-	pillow_python
-	scipy_python
-	setuptools_python
+	haiku${hostArchSuffix}
+	imagemagick${hostArchSuffix}
```

Comment:
```suggestion
	cmd:convert
```
(probably, ImageMagick also provides other commands)

Reply:
cmd:convert is definitely preferred.